packages/react/src/components/ContextualMenu/ContextualMenu.base.tsx

```diff
@@ -725,7 +725,7 @@ class ContextualMenuInternal extends React.Component<IContextualMenuInternalProp
         };
         ariaLabellledby = id;
       } else {
-        const id = sectionProps.title.id || (this._id + sectionProps.title.key.replace(/\s/g, ''));
+        const id = sectionProps.title.id || this._id + sectionProps.title.key.replace(/\s/g, '');
```

Comment:
This was because some incorrect formatting got checked in with another PR. If you merge with master it should be fixed now.